### DISCUSSION TITLE
Can i set a default network?

### DISCUSSION THREAD

**Discussion by FurkanSezal** [rainbow-me/rainbowkit#1450] [Q&A] [answered]
I use multiple wallets in my dapp. Some of them dont support some of networks. If switch between wallets it gives bad UX. What i want to do when a user connect the wallet, i want to connect directly to ethererum mainnet for example. Like a default network. Can i do it? 

My experience if i use metamask in my dapp on bsc network. When i switch the wallet which doesnt support bsc network it doesnt connect dapp without any errors.

**Comment by 0xmkh** [comment, marked as answer]
@FurkanSezal Yes that's possible. You can just add `initialChain` prop to the `<RainbowKitProvider />`. Feel free to follow the docs [here](https://www.rainbowkit.com/docs/chains#customizing-the-initial-chain)
